lib/ProgressPlugin.js

```diff
@@ -66,18 +72,28 @@ const createDefaultHandler = profile => {
 };
 
 class ProgressPlugin {
+	/**
+	 * @param {ProgressPluginArgument} options options
+	 */
 	constructor(options) {
 		if (typeof options === "function") {
 			options = {
 				handler: options
 			};
 		}
+
 		options = options || {};
+		validateOptions(schema, options, "Progress Plugin");
+		options = Object.assign(ProgressPlugin.defaultOptions, options);
+
 		this.profile = options.profile;
 		this.handler = options.handler;
+		this.mode = options.mode;
```

Comment:
There is no reason why not tracking both.

`23/43 entries 102/502 modules ...`

Reply:
As progress value you can use the minimum of both.